oak_restricted_kernel/src/interrupts.rs

```diff
@@ -164,13 +164,164 @@ mutable_interrupt_handler_with_error_code!(
     }
 );
 
+extern "x86-interrupt" fn divide_error_handler(stack_frame: InterruptStackFrame) {
```

Comment:
Are these functions being added to the interrupt handler?
Do we have a custom one?

Reply:
ah, see it on the bottom of the file